public/locales/auset-fr.json

```diff
@@ -0,0 +1,426 @@
+      "setupTitle": "Configurer la Compagnie Riverfolk",
+      "setup": "<i>1.</i> Placez 4 <strong>guerriers</strong> parmi les clairières le long de la rivière.<br/><i>2.</i> Mettez 3 <strong>guerriers</strong> dans votre boîte de paiement.<br/><i>3.</i> Remplissez vos pistes Trade Posts avec les <strong>postes commerciaux</strong> correspondants.<br/><i>4.</i> Mettez votre 3 <strong>marqueurs de service</strong> sur votre piste Services, fixant un prix pour chaque service."
+    },
+    "vagabond": {
+      "name": "Vagabond",
+      "summaryTitle": "Jouer le Vagabond",
+      "summary": "<p>En tant que Vagabond, vous jouerez tous les côtés du conflit, vous faisant des amis et des ennemis comme bon vous semble. Vous marquez des points grâce à vos <strong>relations</strong>, en aidant les factions amies en leur donnant des cartes, et au fur et à mesure que vous développez votre infamie auprès des factions hostiles en retirant leurs pièces au combat. Vous marquerez également des points au cours de vos <strong>quêtes</strong> pour répandre votre réputation parmi les créatures de la forêt.</p> <p>Pour vous déplacer et agir efficacement, vous devrez gérer votre sac d'<strong>objets</strong>, élargir votre sélection en explorant d'anciennes <strong>ruines</strong> et fournir de l'aide à d'autres factions. <strong>Lone Wanderer</strong>, vous ne pouvez pas diriger une clairière ou empêcher une autre faction d'en diriger une, mais vous êtes <strong>Agile</strong>, vous pouvez donc vous déplacer, peu importe qui dirige votre clairière.</p>",
+      "setupTitle": "Configurer le Vagabond",
+      "setup": "<i>1.</i> Placez votre <strong>pion</strong> dans n'importe quelle forêt.<br/><i>2.</i> Mélangez le paquet de quêtes. Piochez 3 <strong>quêtes</strong> à distribuer à proximité.<br/><i>3.</i> Placez le <Sac/>, la <Boot/>, le <Hammer/> et l'<Sword/> <strong>objets de destruction</strong > <i>(marqué d'un\"R\")</i> sous les ruines au hasard, sauf si cela a déjà été fait.<br/><i>4.</i> Mettez le\"{{vagabond}}\" <strong>carte de personnage</strong> dans votre emplacement de carte de personnage. Placez les <InitialStartingItems>, </InitialStartingItems> et <FinalStartingItem/> <strong>éléments de départ</strong> <i>(marqués d'un\"S\")</i> dans votre Cartable et sur leurs pistes correspondantes, si nécessaire.",
```

Comment:
Please remove the extra spaces in the diamond tags, this is causing errors in the rendering of this text.